docs/network/quickstart/qiuck-start.md

```diff
@@ -0,0 +1,215 @@
+---
+title: 'Quick Start'
+sidebar_position: 1
+---
+
+The logstore network can be thought of as a decentralised time-indexed database, meaning essentially that it is a database which can be queried by time. There are several key actions that must be taken in order to be able to store data to the network and retrieve data from the network.
```

Comment:
Let's run this quick start through ChatGPT for punctuation, or at least refine it manually.
ie. "Log Store Network", not "logstore network".

Reply:
Also, please fix the file name...